frontend/src/app/testQueue/TestCard/TestCard.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { useEffect, useRef, useState } from "react";
+import {
+  Card,
+  CardBody,
+  CardHeader,
+  Icon,
+  ModalRef,
+} from "@trussworks/react-uswds";
+import { useNavigate } from "react-router-dom";
+import { useSelector } from "react-redux";
+
+import { DevicesMap, QueriedFacility, QueriedTestOrder } from "../QueueItem";
+import Button from "../../commonComponents/Button/Button";
+import { removeTimer, TestTimerWidget, useTestTimer } from "../TestTimer";
+import { RootState } from "../../store";
+import "./TestCard.scss";
+import TestCardForm from "../TestCardForm/TestCardForm";
+import { useTestOrderPatient } from "../TestCardForm/TestCardForm.utils";
+
+import CloseTestCardModal from "./CloseTestCardModal";
+
+export interface TestCardProps {
+  testOrder: QueriedTestOrder;
+  facility: QueriedFacility;
+  devicesMap: DevicesMap;
+  refetchQueue: () => void;
+  removePatientFromQueue: (patientId: string) => Promise<void>;
+  startTestPatientId: string | null;
+  setStartTestPatientId: React.Dispatch<React.SetStateAction<string | null>>;
+}
+
+export const TestCard = ({
+  testOrder,
+  facility,
+  devicesMap,
+  refetchQueue,
+  removePatientFromQueue,
+  startTestPatientId,
+  setStartTestPatientId,
+}: TestCardProps) => {
+  const navigate = useNavigate();
+  const timer = useTestTimer(
+    testOrder.internalId,
+    testOrder.deviceType.testLength
+  );
+  const organization = useSelector<RootState, Organization>(
+    (state: any) => state.organization as Organization
+  );
+  const closeModalRef = useRef<ModalRef>(null);
+
+  const [isExpanded, setIsExpanded] = useState(true);
+
+  const testCardElement = useRef() as React.MutableRefObject<HTMLDivElement>;
+
+  useEffect(() => {
+    if (startTestPatientId === testOrder.patient.internalId) {
+      testCardElement.current.scrollIntoView({ behavior: "smooth" });
+    }
+    // only run on first render to prevent disruptive repeated scrolls
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const timerContext = {
+    organizationName: organization.name,
+    facilityName: facility!.name,
+    patientId: testOrder.patient.internalId,
+    testOrderId: testOrder.internalId,
+  };
+
+  const { patientFullName, patientDateOfBirth } =
+    useTestOrderPatient(testOrder);
+
+  const toggleExpanded = () => setIsExpanded((prevState) => !prevState);
+
+  const removeTestFromQueue = async () => {
+    await removePatientFromQueue(testOrder.patient.internalId);
+    removeTimer(testOrder.internalId);
+  };
+
+  return (
+    <div
+      data-testid={`test-card-${testOrder.patient.internalId}`}
+      ref={testCardElement}
+    >
+      <CloseTestCardModal
+        closeModalRef={closeModalRef}
+        name={patientFullName}
+        removeTestFromQueue={removeTestFromQueue}
+      />
+      <Card className={"list-style-none margin-bottom-1em test-card-container"}>
```

Comment:
<img width="1350" alt="Screenshot 2023-10-23 at 15 20 24" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/32fc3a34-df3d-4ae3-93ec-1ca5b2e497dd">

Looks like the `<Card>` element is a `<li>` and needs to be nested under `<ul>`

Reply:
Noticed this on the axe scan, but holding off on changing it until the follow-up PR because a few issues popped up with nesting the queue under a single `ul` so additional refactor and updating some tests would be needed